base/libgit2/remote.jl

```diff
@@ -53,6 +87,12 @@ function fetch_refspecs(rmt::GitRemote)
     res
 end
 
+"""
+    push_refspecs(rmt::GitRemote) -> Vector{String}
+
+Get the *push* refspecs for the specified `rmt`. These refspecs contain
```

Comment:
Out of curiosity, why is push bolded? (That's not bad per se, I'm just wondering)